base/src/main/java/com/smartdevicelink/proxy/rpc/SubscribeVehicleData.java

```diff
@@ -515,22 +537,28 @@ public Boolean getInstantFuelConsumption() {
     }
 
     /**
-     * Sets a boolean value. If true, subscribes externalTemperature data
+     * Sets the externalTemperature.
      *
-     * @param externalTemperature a boolean value
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.
+     * @since SmartDeviceLink 2.0.0
+     * @deprecated in SmartDeviceLink 7.1.0
      */
+    @Deprecated
     public SubscribeVehicleData setExternalTemperature(Boolean externalTemperature) {
         setParameters(KEY_EXTERNAL_TEMPERATURE, externalTemperature);
         return this;
     }
 
     /**
-     * Gets a boolean value. If true, means the externalTemperature data has been
-     * subscribed.
+     * Gets the externalTemperature.
      *
-     * @return Boolean -a Boolean value. If true, means the externalTemperature data
-     * has been subscribed.
+     * @return Boolean The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.

```

Comment:
@kostyaBoss On line 557, when I used the RPC Generator tool I got 7.1.0. I would replace the 7.1 with 7.1.0.